content/en/docs/reference/labels-annotations-taints/_index.md

```diff
@@ -256,7 +256,7 @@ Example: `apps.kubernetes.io/pod-index: "0"`
 
 Used on: Pod
 
-When a StatefulSet controller creates a Pod for the StatefulSet, it sets this label on that Pod. 
+When a StatefulSet controller creates a Pod for the StatefulSet, it sets this label on that Pod.
```

Comment:
If you can, avoid making whitespace changes here (and squash down). It helps avoid merge conflicts.

Reply:
Revised.